src/tests/Microsoft.PowerFx.Interpreter.Tests/RecalcEngineTests.cs

```diff
@@ -389,6 +389,53 @@ public void CheckDottedBindError2()
             Assert.StartsWith("Error 2-8: Name isn't valid. 'Value' isn't recognized", result.Errors.First().ToString());
         }
 
+        [Fact]
+        public void CheckDottedBindError3()

```

Comment:
These tests should probably live in BindingEngineTests.cs, since they're not specific to just RecalcEngine.

Reply:
Got it. Moved newly added tests to BindingEngineTests.cs